bin/mpm.json

```diff
@@ -0,0 +1,30 @@
+{
+  "title": "Example JSON Input for MPM",
+  "input_directory" : "/inputFiles/", 
+  "output_directory" : "/results/",
+  "input_files" : {
+    "mesh_filename" : "mesh.smf",
+    "submesh_filename" : "submesh.dat",
+    "constraints_filename" : "mesh.constraints",
+    "soil_particle_filename" : "soilParticles.dat",
+    "init_stress_soil_particle_filename" : "initStressSoilP.dat",
+    "material_filename" : "material.dat",
+    "traction_filename" : "Traction.dat"
+  },
+  "analysis" : {
+    "dt" : 0.001,
+    "number_steps" : 1000,
+    "number_substeps" : 10,
+    "gravity_flag" : 0,
```

Comment:
Flags should be boolean

Reply:
Yes, thus we have 0 or 1? Or do you want "true" or "false"?